types/deploy.go

```diff
@@ -137,6 +128,5 @@ func UpdateDaemonSetTemplateBasedOnStorageNetwork(daemonSet *appsv1.DaemonSet, s
 		}
 	}
 
-	updateHostNetwork()
```

Comment:
Also need to remove https://github.com/longhorn/longhorn-manager/blob/v1.7.0-rc3/types/deploy.go#L136.

Reply:
Thank you. Fixed